NEWS.md

```diff
@@ -5,6 +5,21 @@ Python 3.6 support dropped
 
   afew stopped supporting the older python version 3.6.
 
+Handle DMARC report with empty spf or dkim XML nodes
+
+DMARC Filter: allow to define subject regexp
+
+  Some DMARC report mail subject have a prefix before "Dmarc Report" in
+  the subject and are not checked by the plugin. This commit allows the
```

Comment:
"and are not checked"  -> "and were not checked"

"This commit allows the" <- should be reworded, this is a news page, not a commit.
Maybe "afew now allows… "?

Reply:
Ok I change this part please have a look